API.yaml

```
@@ -183,11 +185,100 @@ paths:
               description: "Directives for caching the response. TEK Chunks are cached for 15 days."
         404:
           description: "TEK Chunk not found."
+  /analytics/apple/operational-info:
+    post:
+      tags:
+      - "Analytics"
+      summary: "Save the operational data in compliance with the monthly policy, avoiding multiple from the same user in a small timeframe, authorized with the provided analytics token."
```

```suggestion
      summary: "Save the operational data sent by the iOS device, in compliance with the monthly policy, authorized with the provided analytics token."
```

I would remove this part as the race condition check is done in the other endpoint.